src/index.html

```diff
@@ -17,6 +17,14 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div
+        class="stopwatch__hand stopwatch__hand--minute stopwatch__hand--minute-speed-up"
+      ></div>
+      <div class="stopwatch__hand-center"></div>
+      <div
+        class="stopwatch__hand stopwatch__hand--second stopwatch__hand--second-speed-up"
+      ></div>
+    </div>
```

Comment:
change your class a bit 
```suggestion
    <div class="stopwatch stopwatch--speed-up">
      <div
        class="stopwatch__hand  stopwatch__minute"
      ></div>
      <div class="stopwatch__hand-center"></div>
      <div
        class="stopwatch__hand stopwatch__second "
      ></div>
    </div>
```